src/core/SampleBuffer.cpp

```diff
@@ -291,11 +291,19 @@ void SampleBuffer::update( bool _keep_settings )
 
 	if( fileLoadError )
 	{
-		QMessageBox::information( NULL,
-				"Fail to open file",
-				"Audio files are limited to 100 MB "
-				"in size and 1 hour of playing time",
+		QString message = "Audio files are limited to 100 MB "
+				"in size and 1 hour of playing time";
+		if( gui )
+		{
+			QMessageBox::information( NULL,
+				"Fail to open file", message,
 					QMessageBox::Ok );
+		}
+		else
+		{
+			fprintf( stderr, "%s\n", message.toUtf8().constData() );
+			exit( EXIT_FAILURE );
```

Comment:
@zonkmachine I think the behavior is inconsistent. With GUI, you can load and render project files with large audio files. However, it's impossible with CLI.
I believe it's okay to remove `exit( EXIT_FAILURE );`.

Reply:
So, if you make a project containing a sample of allowed size and later edit that sample to become large/longer than the limit, exporting from the command line will abort when loading the project instead of just exporting the other tracks... I did that. :clown_face: 

> I believe it's okay to remove `exit( EXIT_FAILURE );`.

:+1:

Reply:
Addressed via 1ab1280843793f16f17d709a3bc47b8c2f5d4e30.